runner/master/run.sh

```diff
@@ -469,6 +508,11 @@ docker run \
 # Copy code in place.
 echo "== Copying code in place"
 docker cp "${CODEDIR}"/. "${WEBSERVER}":/var/www/html
+if [ -v PLUGINSTMPDIR ]
```

Comment:
What is the `-v` check?

I don't see it listed in docs for the builtin `test`

Reply:
`man test`

Reply:
Just confirmed, `-v` is not x-machine compatible.
I'd suggest replacing with:
```
if [ -n "${PLUGINSTOINSTALL}" -a -d "${PLUGINSTMPDIR}" ]
```

Or better still, create a variable like INSTALLPLUGINS="" at the top, and set it to 1 if we do and use:
```
if [ "${INSTALLPLUGINS" ]
```

Reply:
Actually, if we go for my above proposal of making PLUGINDIR always specified, and always exist, then we don't need to put this in an if at all. We always copy the directory because it will always exist.